clj/src/slipstream/ui/views/modal_dialogs.clj

```diff
@@ -141,6 +141,8 @@
                                           (u/enum-append-option :specify-for-each-node))
    :image-target-cloud            (some-> :available-clouds
                                           current-user/configuration)
+   :build-image-target-cloud      (some-> :available-clouds
+                                          current-user/configuration)
    :launch-mutable-run?           false
```

Comment:
Thanks for the comment! It's to have the code similar to lines 139-141. But I'll change it since it does not seem clear. ;)